src/main/java/com/surrealdb/refactor/driver/WsPlaintextConnection.java

```diff
@@ -28,38 +34,41 @@ public class WsPlaintextConnection {
 
     private static final EventLoopGroup group = new NioEventLoopGroup();
     private static final int MAX_CONTENT_LENGTH = 65536;
+
     public WsPlaintextConnection() {}
 
-    public static UnauthenticatedSurrealDB<BidirectionalSurrealDB> connect(URI uri) {
-        Channel channel;
+    public static UnauthenticatedSurrealDB<BidirectionalSurrealDB> connect(final URI uri) {
+        final Channel channel;
         System.out.printf("Connecting to %s\n", uri);
         try {
             channel = bootstrapProtocol(uri).connect(uri.getHost(), uri.getPort()).sync().channel();
-        } catch (InterruptedException e) {
+        } catch (final InterruptedException e) {
             throw new RuntimeException(e);
         }
 
-        return new UnauthenticatedSurrealDB<BidirectionalSurrealDB>() {
+        return new UnauthenticatedSurrealDB<>() {
```

Comment:
Please revert. In the future we are going to turn these into interfaces or abstract classes and it will be good to have the signatures explicit.